faults.csv

```diff
@@ -41,10 +41,7 @@ NRF,No RF ,RF is off,Toggle RF State to On,CAV,,EDM,$EDM/llrf/rf_srf_cavity_main
 NAP,Not in SELAP,Not in SELAP so there is no regulation on amplitude and/or phase,"Verify that there is not active cavity ramp up. If there is not, run auto setup",CAV,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,RFMODE,0,,2,Not in SELAP,
 RFR,RF (not) Ready for Beam,RF not ready for beam: mode is not SELAP and/or RF is off,Run auto setup,CAV,,,,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,,1,,2,RF (not) Ready for Beam,
 CKP,Check Phase,Check Phase: Cavity phase is in an indeterminate state and needs to be measured with the beam before being usable. This fault may appear if something is done to the system that makes 0 phase no longer knowable and ops need to re-gold. This status is set and cleared by experts,"If program permits, phase the cavity",CAV,,SCRIPT,xterm -iconic -T 'SRF Phase Scan GUI xterm' -e run_matlab.bash —matlab=2020a -r SRF_Phasing,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CKPSUM,0,,2,Check Phase,
-PRL,PRL Problem Alarm,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset",RACK,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,PRLSUM.SEVR,,2,2,,
-PRL,PRL Problem Alarm,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset",RACK,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,PRLSUM.SEVR,,2,2,PRL Problem,
-PRL,PRL Problem Warning,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset",RACK,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,PRLSUM.SEVR,,1,1,,
-PRL,PRL Problem Warning,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset",RACK,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,PRLSUM.SEVR,,1,1,PRL Problem,
+PRL,PRL Problem Alarm,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset",CAV,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PRLSUM.SEVR,,2,2,PRL Problem,

```

Comment:
```suggestion
PRL,PRL Problem Alarm,Problem with Phase Reference. Cavity phase may not be locked to global RF.,"If the program permits and there is RF in the cavity, PRL Tracking > Accept Current Offset. This will put the cavity into CKP and make a re-phasing necessary.",CAV,,EDM,$EDM/llrf/rf_srf_prl_cm.edl,"SELTAB=8,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PRLSUM.SEVR,,2,2,PRL Problem,

```